Demo/Fullscreen/FavoriteAdsListView/FavoriteAdsListDemoViewHelper.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import FinniversKit
+
+struct FavoriteAd: FavoriteAdViewModel {
+    let addressText: String?
+    let titleText: String
+    let descriptionPrimaryText: String?
+    let descriptionSecondaryText: String?
+    let imagePath: String?
+    let ribbonStyle: RibbonView.Style
+    let ribbonTitle: String
+    let addedToFolderDate: Date
+}
+
+struct FavoriteAdsFactory {
+    static func create() -> [FavoriteAd] {
+        var favorites = [FavoriteAd]()
+        for (index, title) in titles.enumerated() {
```

Comment:
Could also do it with `titles.enumerated().map`.

Reply:
🙌